src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+
+import './CommentList.scss';
+
+interface CommentListProps {
+  comments: Comment[]
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0
```

Comment:
You can make it simple
```suggestion
    {comments.length
```